app/views/templates/root_node.html.haml

```diff
@@ -2,17 +2,45 @@
 
 %article.content-listing
   %section.home-category-list
-    %h2 Information and services
+    %h2 What is GOV.AU Beta?
+    .abstract
+      %p
+        GOV.AU Beta is a small imagining of what a whole-of-government platform could look like. Its goal is to make Australian Government information and services simpler, clearer and faster.
```

Comment:
It would be nicer to word wrap these lines to 100 characters. Just for readability.